src/main/resources/Shops.json

```diff
@@ -1,8 +1,14 @@
 [
   {
-    "name": "Name",
+    "name": "generalStoreLumbrige",
```

Comment:
Do general stores stock different items per location? if not can shorten to just one general store variable.
(or just have a extra if just some general stores have different stock.

Reply:
Yeah stock varies between general stores. Some stores should be the same which should save on file length.